src/net_processing.cpp

```diff
@@ -867,6 +907,13 @@ void PeerLogicValidation::ReattemptInitialBroadcast(CScheduler& scheduler) const
 }
 
 void PeerLogicValidation::FinalizeNode(NodeId nodeid, bool& fUpdateConnectionTime) {
+    int misbehavior;
```

Comment:
36ac41a seems safer to initialize `misbehavior` with a value, even if set ATM a few lines further down

Reply:
ok. Done.